jeremy2918/tsseract
# Added Material UI dependencies

# [Material UI](https://trello.com/c/RRoDGioS/26-i-want-to-add-material-ui-as-the-front-end-library-for-the-project)

## Description
This branch adds the [Material UI](https://material-ui.com/) libraries to develop the front end of the app. It also created a theme and add the typography fonts to the project.


## Testing Steps

- `git pull`
- `git checkout materialUI`
- `npm install`
- `npm run tsc`
- `npm run dev:server`
- The page should continue working as usual.



## Changes
No UI changes
